plover/config.py

```diff
@@ -383,7 +396,18 @@ def get_translation_frame_y(self):
         return self._get_int(TRANSLATION_FRAME_SECTION, 
                              TRANSLATION_FRAME_Y_OPTION,
                              DEFAULT_TRANSLATION_FRAME_Y)
-                             
+
+    def set_translation_frame_opacity(self, opacity):
```

Comment:
@benoit-pierre has suggested just asserting the function input instead of correcting it, especially because this is an internal-facing call.